src/InstrumentationEngine.Attach/CommandHandler/AttachCommandHandler.cs

```diff
@@ -233,9 +233,12 @@ internal class AttachCommandHandler
             }
 
             XmlSerializer engineConfigSerializer = new XmlSerializer(typeof(InstrumentationEngineConfiguration));
-            MemoryStream memStream = new MemoryStream();
-            engineConfigSerializer.Serialize(memStream, configuration);
-            byte[] bytes = memStream.ToArray();
+            byte[] bytes;
```

Comment:
>byte[] bytes; [](start = 12, length = 13)

?

Reply:
I didn't want to extend the MemoryStream IDisposable scope until the end of the method, so I declared the bytes array like this.